website/mails.py

```diff
@@ -180,7 +180,7 @@ def send_mail(to_addr, mail, mimetype='plain', from_addr=None, mailer=None,
     subject='Your ${action} has failed',
 )
 
-UNESCAPE = "<% from website.util.sanitize import safe_unescape_html %> ${safe_unescape_html(src.title)}"
+UNESCAPE = "<% from website.util.sanitize import unescape_entities %> ${unescape_entities(src.title)}"
```

Comment:
It seems a bit weird to me that a python file has embedded Mako logic in a string; possible separation of concerns issue? Probably outside scope for this PR though.